server/search/userController.js

```diff
@@ -20,28 +20,28 @@ module.exports = function(body, res){
   var userObj = {};
 
   async.forEachOf(itemGroup, function (value, key, callback) {
-    gitHTTP('GET', itemGroup[key].contributors_url + '?', function(err, response, contributors){
+    gitHTTP('GET', itemGroup[key].contributors_url + '?', function(err, response, contributor){
```

Comment:
why bother changing this? you can var set a local variable and take scope away from a callback argument